CMakeLists.txt

```diff
@@ -9,7 +9,8 @@ execute_process(
 )
 
 set(SKELLYSIM_VERSION "0.9.12")
-message("SkellySim version: v${SKELLYSIM_VERSION} (${SKELLYSIM_COMMIT})")
+set(SKELLYSIM_TRAJECTORY_VERSION "1")
```

Comment:
Feel like just defining this via header is better. I've had issues with caching screwing this up and getting mixed defines. Not a big issue for version/commit, but would break things otherwise.

Reply:
I'm going to leave this one for now, and then come back to it at some point in the future, and you can explain what you're thinking.